bgpd/bgp_evpn_vty.c

```diff
@@ -5950,6 +5943,16 @@ DEFUN (bgp_evpn_vrf_rt,
 		return CMD_WARNING_CONFIG_FAILED;
 	}
 
+	if (rt_type == RT_TYPE_BOTH || rt_type == RT_TYPE_EXPORT) {
+		for (int i = 2; i < argc; i++) {
```

Comment:
Wouldn't be better to handle this in parse_rtlist() and return it as an argument `is_wildcard`? This way, we don't need this loop.

Reply:
"33:44 ... *:46", the wildcard RT maybe is in the last.
If using `parse_rtlist()` existing loop, it need to `free` other RTs. It is unnecessary, so i didn't do that way.